ni_measurement_plugin_sdk_service/_internal/parameter/decoder_strategy.py

```diff
@@ -245,7 +150,17 @@ def get_decoder(
 
 def get_type_default(type: type_pb2.Field.Kind.ValueType, repeated: bool) -> Any:
     """Get the default value for the give type."""
+    _type_default_mapping = {
+        type_pb2.Field.TYPE_FLOAT: float(),
+        type_pb2.Field.TYPE_DOUBLE: float(),
+        type_pb2.Field.TYPE_INT32: int(),
+        type_pb2.Field.TYPE_INT64: int(),
+        type_pb2.Field.TYPE_UINT32: int(),
+        type_pb2.Field.TYPE_UINT64: int(),
+        type_pb2.Field.TYPE_BOOL: bool(),
+        type_pb2.Field.TYPE_STRING: str(),
+        type_pb2.Field.TYPE_ENUM: int(),
+    }
     if repeated:
         return list()
-    type_default_value = _TYPE_DEFAULT_MAPPING.get(type)
-    return type_default_value
+    return _type_default_mapping.get(type)
```

Comment:
I moved it there to make everything under one function, I wasn't worried about avoiding any bugs or anything like that. Although I don't think I need to use copy.copy to avoid that bug so I'll move it above the function and leave it as that, is that ok?